client/wp-admin-scripts/print-shipping-label-banner/shipping-banner/index.js

```diff
@@ -300,40 +300,61 @@ export class ShippingBanner extends Component {
 
 			const wcsStoreUnsubscribe = wcsStore.subscribe( () => {
 				const latestState = wcsStore.getState();
+
 				const siteState =
 					latestState.extensions.woocommerce.woocommerceServices[
 						siteId
 					];
 				if ( siteState ) {
 					const orderState = siteState.shippingLabel[ orderId ];
 					if ( orderState ) {
-						if ( orderState.showPurchaseDialog ) {
+						if (
+							! orderState.showPurchaseDialog &&
+							! orderState.isFetching
+						) {
+							wcsStore.dispatch( {
+								type:
+									'WOOCOMMERCE_SERVICES_SHIPPING_LABEL_OPEN_PRINTING_FLOW',
+								orderId,
+								siteId,
+							} );
+							wcsStore.dispatch( {
+								type: 'NOTICE_CREATE',
+								notice: {
+									duration: 10000,
+									status: 'is-success',
+									text: __(
+										'Plugin installed and activated',
+										'woocommerce-admin'
+									),
+								},
+							} );
+						} else if ( orderState.showPurchaseDialog ) {
 							if ( window.jQuery ) {
 								window
 									.jQuery( '#woocommerce-order-label' )
 									.show();
 							}
 							wcsStoreUnsubscribe();
```

Comment:
These actions are the ones triggered when the user press "Use address as entered" in the address step of the create shipping label modal.